src/Place/Components/PCO002GooglePlacesAutocomplete/GooglePlacesAutocomplete.tsx

```diff
@@ -0,0 +1,40 @@
+import { PlaceAutocompleteResult } from '@googlemaps/google-maps-services-js';
+import { AutocompleteInput } from 'react-native-autocomplete-input';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+import { Text } from 'react-native-paper';
+
+import { PCO002GooglePlacesAutocompleteController } from './GooglePlacesAutocompleteController';
+import { GooglePlacesAutocompletePropsInterface } from './GooglePlacesAutocompletePropsInterface';
+
+import i18n from '@/Common/Hooks/i18n-js';
+
+export const PCO002GooglePlacesAutocomplete = ({
+	// TODO: 「'onAutoCompleteClicked' is missing in props validation」ESlintErrorが出る(onlySpot, fetchDetailsも)
+	onAutoCompleteClicked,
+	onlySpot,
+	fetchDetails,
+}: GooglePlacesAutocompletePropsInterface) => {
+	const { searchInput, placesResult, onChangeInput, onPressAutocomplete } = PCO002GooglePlacesAutocompleteController({
+		onAutoCompleteClicked,
+		onlySpot,
+		fetchDetails,
+	});
+
+	return (
+		<AutocompleteInput
+			data={placesResult}
+			value={searchInput}
+			onChangeText={onChangeInput}
+			flatListProps={{
+				// TODO: 「'item' PropType is defined but prop is never used」eslintエラーが出る
+				renderItem: ({ item }: { item: PlaceAutocompleteResult }) => (
```

Comment:
なんか↓みたいに修正したらこのファイルのESLintは全部消えた（謎）
```suggestion
				renderItem: ({ item }: ListRenderItemInfo<PlaceAutocompleteResult>) => (
```

Reply:
[f113e9f](https://github.com/Ayato-kosaka/spelieve/pull/390/commits/f113e9fa2ba7e002de13d662712f878723a4e8e8)で対応